tests/unit/core/reducers/test_addons.js

```diff
@@ -82,4 +82,38 @@ describe('addon reducer', () => {
       },
     });
   });
+
+  it('does not use description from theme_data', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/2569
+    // TODO: Remove this when
+    // https://github.com/mozilla/addons-frontend/issues/1416 is fixed.
+    const type = ADDON_TYPE_THEME;
+    const state = addons(originalState, {
+      payload: {
+        entities: {
+          addons: {
+            baz: {
+              description: null,
+              slug: 'baz',
+              id: 42,
+              theme_data: { theme_thing: 'some-data', description: 'None' },
+              type,
+            },
+          },
+        },
+      },
+    });
+    expect(state).toEqual({
```

Comment:
Agreed, just doing what was already done but I guess I'm refactoring the tests anyway 😉